main.go

```diff
@@ -1402,8 +1402,12 @@ func (tf *transformer) loadPkgCache(files []*ast.File) error {
 	ssaPkg := ssaProg.CreatePackage(tf.pkg, files, tf.info, false)
 	ssaPkg.Build()
 
-	tf.reflectCheckedAPIs = make(map[string]bool)
-	tf.recordReflection(ssaPkg)
+	inspector := reflectInspector{
+		pkg:         tf.pkg,
+		checkedAPIs: make(map[string]bool),
+		result:      curPkgCache, // append the results
```

Comment:
a little unusual way to store results for me

Reply:
I want `reflectInspector` to be able to store its results somewhere that isn't always the same global, that's what I meant to do with this `result` field. An alternative would be to pass around a parameter or result in all methods, but then all the method types and calls get significantly more verbose.

Do you have anything else in mind?

Reply:
I re-read code and think this is best variant, pass as a parameter everywhere variant is much worse.